mix.exs

```diff
@@ -13,7 +13,7 @@ defmodule CredoFilenameConsistency.MixProject do
       homepage_url: "https://github.com/mirego/credo_filename_consistency",
       docs: [extras: ["README.md"], main: "readme", source_ref: "v#{@version}", source_url: "https://github.com/mirego/credo_filename_consistency"],
       package: package(),
-      elixir: "~> 1.8",
+      elixir: "~> 1.7",
```

Comment:
If we want to relax the version number, we should also test both version (1.7 and 1.8) on Travis 😄

Reply:
I'll do this when I get home from beer tonight. :-)

Reply:
Done. Turns out 1.6 is fine as well. :-)